programs/clearing_house/src/controller/orders.rs

```diff
@@ -420,6 +420,17 @@ pub fn fill_order(
     let (order_status, market_index) =
         get_struct_values!(user.orders[order_index], status, market_index);
 
+    // settle lp position so its tradeable
+    {
+        let market = &mut market_map.get_ref_mut(&market_index)?;
```

Comment:
should we call this every where we call `settle_funding_payment`?

Reply:
rn the fcns that have settle_funding but not settle_lp is sanitize maker order, liquidation (happens through burn tho), and settle_pnl -- i think adding it to settle_pnl is a good idea, the others i think we're g